docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -705,6 +785,103 @@ The ``FV3LAM_wflow.xml`` file runs the specific j-job scripts (``regional_workfl
    | run_post             | Run the post-processing tool (UPP)                         |
    +----------------------+------------------------------------------------------------+
 
+In addition to the baseline tasks described in :numref:`Table %s <WorkflowTasksTable>` above, users may choose to run some or all of the METplus verification tasks. These tasks are described in :numref:`Table %s <VXWorkflowTasksTable>` below. 
+
+.. _VXWorkflowTasksTable:
+
+.. table:: Verification (VX) workflow tasks in the SRW App
+
+   +-----------------------+------------------------------------------------------------+
+   | **Workflow Task**     | **Task Description**                                       |
+   +=======================+============================================================+
+   | GET_OBS_CCPA          | Retrieves and organizes hourly :term:`CCPA` data from NOAA |
```

Comment:
Is it worth mentioning in this table for the GET_OBS tasks that they are only an option (to set to TRUE) if the user has access to NOAA HPSS?